README.md

```diff
@@ -1 +1,5 @@
-# Review-project-from-trials
\ No newline at end of file
+# FCC Test Suite Template
+
```

Comment:
You should add the Getting Started section and its sub-section

Reply:
- [ ]  Always make sure to include an explanation, a "why", for each requested change, and use polite words. Kindly edit and submit the project.